pkg/client/didexchange/client.go

```diff
@@ -153,12 +153,10 @@ func (c *Client) HandleInvitation(invitation *Invitation) error {
 }
 
 // QueryConnections queries connections matching given parameters
-func (c *Client) QueryConnections(request *QueryConnectionsParams) ([]*Connection, error) {
-	// TODO https://github.com/hyperledger/aries-framework-go/issues/429 query all connections from did exchange service
-	return []*Connection{
-		{&didexchange.ConnectionRecord{ConnectionID: uuid.New().String()}},
-		{&didexchange.ConnectionRecord{ConnectionID: uuid.New().String()}},
-	}, nil
+func (c *Client) QueryConnections(request *QueryConnectionsParams) ([]*didexchange.ConnectionRecord, error) {
```

Comment:
We shouldn't expose service pkg(didexchange.ConnectionRecord) to consumers/business logic(bdd/rest).

Reply:
Yes I am fixing it in upcoming PR with bddtests. This one I pushed in hurry for demo. ALso, I see some swagger annotations used in client models which I am going to fix as well.